src/py/flwr/server/grpc_server/driver_client_manager.py

```diff
@@ -72,7 +72,7 @@ def register(self, client: ClientProxy) -> bool:
             return False
 
         # Generate random integer ID
-        random_node_id: int = uuid.uuid1().int >> 64
+        random_node_id: int = random.randrange(9223372036854775806)
```

Comment:
The max value is 9223372036854775807, so shouldn't this be:
```suggestion
        random_node_id: int = random.randrange(9223372036854775808)
```